src/createNavigationContainer.js

```diff
@@ -153,9 +155,11 @@ export default function createNavigationContainer<T: *>(
         return;
       }
 
-      this.subs = BackAndroid.addEventListener('backPress', () =>
-        this.dispatch(NavigationActions.back())
-      );
+      if (hardwareBackHandled !== false) {
```

Comment:
For props name, how about `backAndroidHandled`to make it consistent with next line `BackAndroid.addEventListener`?